tests/sdk/java/testdata/await/generic/index.ts

```diff
@@ -0,0 +1,44 @@
+import * as kubernetes from "@pulumi/kubernetes";
+
+const ns = new kubernetes.core.v1.Namespace("ns", {
+  metadata: {
+    name: "generic-await",
+  },
+});
+
+const provider = new kubernetes.Provider("provider", {
+  namespace: ns.metadata.name,
+});
+
+const crds = new kubernetes.yaml.v2.ConfigFile(
+  "crds",
+  { file: "crds.yaml" },
+  { provider: provider }
+);
+
+export const wantsReady = new kubernetes.apiextensions.CustomResource(
```

Comment:
If you used nodejs instead of yaml runtime solely to be able to create a custom resource, another option is to use `ConfigGroup` V2 because it also can create custom resources, in yaml.

Oh, seems you have a yaml alternative below, why both?